options.go

```diff
@@ -31,3 +32,12 @@ var DefaultOptions = &Options{
 	AutomaticHostHeader:    true,
 	AutomaticContentLength: true,
 }
+
+func init() {
+	fd, err := fastdialer.NewDialer(fastdialer.DefaultOptions)
```

Comment:
`fastdialer.DefaultOptions` is disk based, I think we need to handle cleanup properly (`fastdialerInstance.Close()`) on exit to avoid cases like https://github.com/projectdiscovery/nuclei/issues/3693

Reply:
@Mzack9999 , since we using global insance of fastdialer here that might not be possible.
i will move logic to client and if fastdialer is nil when creating new rawhttp [client](https://github.com/projectdiscovery/rawhttp/blob/main/client.go), new instance  of fastdialer is created  and also add a method `.Close()` where we can close fastdialer and any other resources. what do you think ?

Reply:
Valid solution, as an alternative we might consider in the future to use memory type storage